src/content/reference/react/useMemo.md

```diff
@@ -1105,7 +1105,7 @@ function ChatRoom({ roomId }) {
     const connection = createConnection(options);
     connection.connect();
     return () => connection.disconnect();
-  }, [options]); // ✅ Only changes when createOptions changes
+  }, [options]); // ✅ Only changes when create options changes
```

Comment:
Maybe it should be "Only changes when options changes"? What is create options?

Reply:
Additionally, line 1104 `const options = createOptions();` should be removed.

Reply:
ok，thankyou

Reply:
Please check #7335
@ZhenyaGoroh @lukahartwig 
Thank you all

Reply:
> Please check #7335 @ZhenyaGoroh @lukahartwig Thank you all

I left the comment. Also I'd suggest to commit changed to this pr instead of creating a new one.